GUIDE.rst

```diff
@@ -209,26 +211,17 @@ define our handlers as coroutines and yield to IO operations:
 
 .. code-block:: python
 
-    @app.register(KeyValue)
+    @tchannel.register(KeyValue)
     @gen.coroutine
-    def setValue(request, response):
-        key = request.args.key
-        value = request.args.value
+    def setValue(request):
+        key = request.body.key
+        value = request.body.value
```

Comment:
I would make it `args = request.body` and then `args.key` and `args.value`. I want to make it clear that the method parameters are what you get in the `request.body`